change/@griffel-webpack-extraction-plugin-b04e7a25-7497-4ba1-8e78-2e4163fc33e7.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat(webpack-extraction-plugin): add NextJS plugin",
```

```suggestion
  "comment": "feat: add plugin for NextJS",
```

nit: As this will go to a changelog of the package there is no reason to have it there.